src/main/java/org/programmers/springbootbasic/voucher/model/PercentDiscountVoucher.java

```diff
@@ -0,0 +1,32 @@
+package org.programmers.springbootbasic.voucher.model;
+
+import java.util.UUID;
+
+public class PercentDiscountVoucher implements Voucher{
+    private final UUID voucherId;
+    private final long percent;
+
+    public PercentDiscountVoucher(UUID voucherId, long percent) {
+        if (percent < 0) throw new IllegalArgumentException("할인율은 0보다 커야 합니다.");
+        if (percent == 0) throw new IllegalArgumentException("할인율은 0이 아니어야 합니다.");
+        if (percent > 99) throw new IllegalArgumentException("할인율은 100보다 작아야 합니다.");
```

Comment:
```suggestion
	if(isNotRangePercent(percent)) {
		throw new IllegalArgumentException("할인율은 0 초과 100 이하여야 합니다.");
	}
```

예외 한번에 처리가 가능하지 않을까요 !?